ksql-engine/src/main/java/io/confluent/ksql/engine/TopicAccessValidatorFactory.java

```diff
@@ -28,15 +28,23 @@
 public final class TopicAccessValidatorFactory {
   private static final Logger LOG = LoggerFactory.getLogger(TopicAccessValidatorFactory.class);
   private static final String KAFKA_AUTHORIZER_CLASS_NAME = "authorizer.class.name";
+  private static final TopicAccessValidator DUMMY_VALIDATOR = (sc, metastore, statement) -> { };
 
   private TopicAccessValidatorFactory() {
-
   }
 
   public static TopicAccessValidator create(
-      final ServiceContext serviceContext,
-      final MetaStore metaStore
+      final KsqlConfig ksqlConfig,
+      final ServiceContext serviceContext
   ) {
+    final String enabled = ksqlConfig.getString(KsqlConfig.KSQL_ENABLE_TOPIC_ACCESS_VALIDATOR);
+    if (enabled.equals(KsqlConfig.KSQL_ACCESS_VALIDATOR_ON)) {
```

Comment:
Optional:
Is case-sensitive a requirement in all KSQL configurations? Should we allow the use of any case?

Reply:
The only other similar config (`ksql.named.internal.topics`) is case-sensitive. I don't feel particularly strongly one way or another.